.github/CODEOWNERS

```
@@ -1,4 +1,4 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
 # Primary repo maintainers
-*       @noot @arijitAD @edwardmack @timwu20 @EclesioMeloJunior @jimjbrettj @kishansagathiya @qdm12
\ No newline at end of file
+*       @noot @arijitAD @edwardmack @timwu20 @EclesioMeloJunior @jimjbrettj @kishansagathiya @qdm12 @dimartiro @kanishkatn
```

I notice this error in the review:
```
Unknown owner on line 4: make sure @arijitAD exists and has write access to the repository
```

I think we should remove @arijitAD 😢 
```suggestion
*       @noot @edwardmack @timwu20 @EclesioMeloJunior @jimjbrettj @kishansagathiya @qdm12 @dimartiro @kanishkatn
```

---

Yeah I think this is a good point, sad but will update